proposals/Level-1.md

```diff
@@ -143,19 +141,13 @@ block.
 
 In the initial implementation, try blocks may only yield 0 or 1 values.
 
-### Exception creation
+### Throwing an exception
 
-A `except` instruction has a single immediate argument, an exception tag.  The
-corresponding tagged exception type is used to define the data fields of the
-created exception. The values for the data fields must be on top of the operand
-stack, and must correspond to the exception's type signature.  These values are
-popped off the stack and an instance of the exception is then created. A
-reference to the created exception is then pushed onto the stack.
-
-### Throws
-
-The `throw` throws the exception on top of the stack. The exception is popped
-off the top of the stack before throwing.
+The `throw` instruction takes an exception index as an immediate argument.  That
+index is used to identify the tagged exception type to throw.  The values to
+throw (corresponding to the signature) are popped from the top of the stack, and
+an exception is created. This includes attaching a stack track to the exception.
```

Comment:
Nit: The stack trace isn't part of the core semantics and its presence may depend on the embedder.

Reply:
Removed.